css-two-column-layout/styles.css

```diff
@@ -0,0 +1,98 @@
+* {
+  font-family: "Karla", sans-serif;
+  box-sizing: border-box;
+}
+
+.container {
+  width: 800px;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+h1 {
+  letter-spacing: 0.25rem;
+  margin-bottom: 0;
+}
+
+.top-paragraph {
+  width: 58%;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 11px;
+  line-height: 1.25rem;
+  color: gray;
+}
+
+h1,
+h4 {
+  text-transform: uppercase;
+}
+
+h4 {
+  text-align: left;
+  font-size: small;
+  margin-bottom: 1px;
+}
+
+.paragraph {
+  color: gray;
+  text-align: start;
+  font-size: x-small;
+  line-height: 1.2rem;
+}
+
+.row {
+  display: flex;
+  width: 90%;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-top: 30px;
+  margin-bottom: 0;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-half {
+  display: flex;
```

Comment:
If you are applying `display: flex` to a column class, that usually means that the `column` should contain a `row` class element. That is the purpose of the `row` class, to make flex containers